sdk/iot/Azure.Iot.Hub.Service/tests/CustomRequestSanitizer.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+
+namespace Azure.Iot.Hub.Service.Tests
+{
+    /// <summary>
+    /// Custom sanitizer to remove secrets from the recorded json files used for playback tests.
+    /// </summary>
+    internal class CustomRequestSanitizer : RecordedTestSanitizer
+    {
+        internal const string FAKE_HOST = "FakeHost.net";
```

Comment:
Is this our constant casing standard? Should be `FakeHost` right?

Reply:
I just renamed this file. We already had it checked in that way. I have seen both kinds of constants. I usually go with CamelCasing but did not change this. I don't mind making the change.

Reply:
Let's make it consistent. Thanks